gradle/libs.versions.toml

```diff
@@ -25,6 +25,7 @@ terminalconsoleappender = "1.2.0"
 folia = "1.19.4-R0.1-SNAPSHOT"
 viaversion = "4.0.0"
 adapters = "1.9-SNAPSHOT"
+cloud = "1.8.3"
```

Comment:
Just FYI latest release is 1.8.4
https://github.com/Incendo/cloud/releases/tag/1.8.4

Reply:
oh yippee i'm a part of that